src/magic_folder/config.py

```diff
@@ -40,6 +50,11 @@
     atomic_makedirs,
 )
 
+# Export this here since GlobalConfigDatabase is what it's for.
+from ._endpoint_parser import (
+    endpoint_description_to_http_api_root,
+)
+
 
```

Comment:
Would it make sense to add an additional helper in `GlobalConfigDatabase`, like this:

```python
    def http_api_root(self):
        """
        Base URL for HTTP API, derived from endpoint describtion.
        """
        return endpoint_description_to_http_api_root(self.api_endpoint)
```
Perhaps consumers of the code in `config.py` can avoid some duplication that way?

Reply:
It might make sense, I'm not sure.  It might also come down to taste.  I almost made it a method but decided against it based on the idea that GlobalConfigDatabase's job is to allow *state* to be stored and retrieved, not to provide all of the different convenient interfaces to that state.  Adding one accessor is hardly a terrible burden but maybe it's a step in the direction we don't want to go, so I'm inclined to leave it off for now and then add it later if it turns out it would be a significant benefit (we could also consider adding the above except as a free function instead of a method which might be 99% of the convenience without expanding `GlobalConfigDatabase`'s interface.

Reply:
That makes sense.  Thank you.